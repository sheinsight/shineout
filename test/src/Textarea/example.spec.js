import exampleTest from '../../example'

describe('Textarea[snapshot]', () => {
  exampleTest('Textarea')
})
