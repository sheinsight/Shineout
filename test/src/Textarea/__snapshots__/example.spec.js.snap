// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test: Textarea[site/pages/components/Textarea/example-1-base.js] 1`] = `
<label>
  <textarea />
</label>
`;

exports[`Snapshot test: Textarea[site/pages/components/Textarea/example-2-autosize.js] 1`] = `
<div>
  <label>
    <textarea />
    <textarea />
  </label>
  <br />
  <label>
    <textarea />
    <textarea />
  </label>
</div>
`;

exports[`Snapshot test: Textarea[site/pages/components/Textarea/example-3-info.js] 1`] = `
<label>
  <textarea />
</label>
`;

exports[`Snapshot test: Textarea[site/pages/components/Textarea/example-4-custom.js] 1`] = `
<label>
  <textarea />
</label>
`;
